src/bin/vip-logs.js

```diff
@@ -243,7 +249,7 @@ command( {
 		`The maximum number of entries to return. Accepts an integer value between 1 and 5000 (defaults to ${ LIMIT_DEFAULT }).`
 	)
 	.option( 'follow', 'Output new entries as they are generated.' )
-	.option( 'format', 'Render output in a particular format. Accepts “csv”, and “json”.', 'table' )
+	.option( 'format', 'Render output in a particular format. Accepts “csv”, “json”, and “text”.', 'table' )
```

Comment:
```suggestion
	.option( 'format', 'Render output in a particular format. Accepts “csv”, “json”, “table”, and “text”.', 'table' )
```

@yolih what do you think?